netebpfext/net_ebpf_ext.h

```diff
@@ -19,6 +19,7 @@
 #include "framework.h"
 #include "netebpfext_platform.h"
 
+#include <NtStrsafe.h>
```

Comment:
Does this need to be done before the other headers below?
List in alphabetical order if not.
But if there is an ordering dependency add a // comment saying what it has to be before.

Reply:
Yes, there is a dependency on ntddk.h (included in framework.h).  I did think of adding this header there, but thought it better to scope it to where it was really needed (i.e. the NetEBpfExt project), so included it here instead.

It's all moot now anyway, as I don't need this header anymore (don't need the RtlPrintfxxx call anymore).